graphql-schemas/schema.yml

```diff
@@ -899,7 +899,9 @@ confs:
 - name: OpenShiftClusterManagerRecommendedVersions_v1
   fields:
   - { name: recommendedVersion, type: string, isRequired: true }
+  - { name: initialVersion, type: string, isRequired: true }
```

Comment:
i am not sure of the wording here. The version should be the same as the `recommendedversion`. Only its format will be different, right? I think we mentioned `ocmVersion` or something like that, didn't we?

Reply:
Funny, I remembered initial version, cause we use it in App-Interface ourselves.